src/components/shared/table-filters/__tests__/FacetedMultiSelectFilter.test-utils.tsx

```
@@ -0,0 +1,96 @@
+  children: React.ReactNode
+  onClick?: (...args: unknown[]) => void
+} & Record<string, unknown>
+
+type PopoverContentProps = PopoverStateProps & {
+  children: React.ReactNode
+} & React.HTMLAttributes<HTMLDivElement>
+
+vi.mock("@heroui/react/input", () => ({
+  Input: React.forwardRef<HTMLInputElement, React.InputHTMLAttributes<HTMLInputElement>>(
+    (props, ref) => <input {...props} ref={ref} />
+  ),
```

ESLint

[error] Component definition is missing display name

(react/display-name)